src/tests/Microsoft.VisualStudio.Composition.Tests/RuntimeCompositionTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.VisualStudio.Composition.Tests
+{
+    using System;
+    using Xunit;
+
+    public class RuntimeCompositionTests
```

Comment:
Can you also add a test for the 2 `ArgumentException` cases that you now throw?

Reply:
Done.